app/uk/gov/hmrc/hec/services/scheduleService/HecTaxCheckExtractionService.scala

```diff
@@ -19,59 +19,73 @@ package uk.gov.hmrc.hec.services.scheduleService
 import cats.data.EitherT
 import cats.implicits._
 import com.google.inject.{ImplementedBy, Inject}
-import play.api.Configuration
 import uk.gov.hmrc.hec.models
-import uk.gov.hmrc.hec.models.HECTaxCheck
-import uk.gov.hmrc.hec.services.TaxCheckService
+import uk.gov.hmrc.hec.models.licence.{LicenceTimeTrading, LicenceType, LicenceValidityPeriod}
+import uk.gov.hmrc.hec.models.{Error, HECTaxCheck}
+import uk.gov.hmrc.hec.services.{FileCreationService, FileStoreService, MongoLockService, TaxCheckService}
 import uk.gov.hmrc.hec.util.Logging
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.mongo.lock.{LockService, MongoLockRepository}
 
 import javax.inject.Singleton
 import scala.concurrent.Future
-import scala.concurrent.duration.FiniteDuration
 
 @ImplementedBy(classOf[HecTaxCheckExtractionServiceImpl])
 trait HecTaxCheckExtractionService {
 
-  def lockAndExtractJob(): Future[Option[Either[models.Error, List[HECTaxCheck]]]]
+  def lockAndProcessHecData(): Future[Option[Either[models.Error, List[HECTaxCheck]]]]
 
 }
 
 @Singleton
 class HecTaxCheckExtractionServiceImpl @Inject() (
   taxCheckService: TaxCheckService,
-  mongoLockRepository: MongoLockRepository,
-  config: Configuration
+  mongoLockService: MongoLockService,
+  fileCreationService: FileCreationService,
+  fileStoreService: FileStoreService
 )(implicit
   hecTaxCheckExtractionContext: HECTaxCheckExtractionContext
 ) extends HecTaxCheckExtractionService
     with Logging {
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
 
-  val lockService: LockService = LockService(
-    mongoLockRepository,
-    lockId = "hecTaxChecks",
-    ttl = config.get[FiniteDuration]("mongo-lock.force-lock-release-after")
-  )
+  val licenceType: (String, String)           = ("licence-type", "LICENCE_TYPE")
```

Comment:
it'd be worth having a mini case class maybe in the companion object to `HecTaxCheckExtractionServiceImpl` to stick these strings in so it's clearer what they actually are